packages/tailwind/src/utils/get-css-for-markup.ts

```diff
@@ -0,0 +1,47 @@
+import postcss from "postcss";
+import tailwindcss from "tailwindcss";
+import type { CorePluginsConfig } from "tailwindcss/types/config";
+import postcssCssVariables from "postcss-css-variables";
+import type { TailwindConfig } from "../tailwind";
+
+declare global {
+  // eslint-disable-next-line no-var
+  var __OXIDE__: undefined;
+}
+
+global.__OXIDE__ = undefined;
+// to avoid __OXIDE__ undefined errors
+// this may cause problems later down the line when upadting tailwind
+// since tailwind might migrate to using oxide for their transformations
+
+export function getCSSForMarkup(
```

Comment:
can you use arrow function and change the casing?

```suggestion
export function getCssForMarkup(
```